### ISSUE TITLE
eslint now in 10.0

### ISSUE THREAD

**Issue by steve02081504** [benmosher/eslint-plugin-import#3265]
```
PS C:\Users\steve02081504\workstation\fount> deno run -A npm:eslint --quiet .
Warning The following peer dependency issues were found:
└─┬ eslint-plugin-import@2.32.0 (imported by https://cdn.jsdelivr.net/gh/steve02081504/my-eslint-config/deno.mjs)
  └── peer eslint@^2 || ^3 || ^4 || ^5 || ^6 || ^7.2.0 || ^8 || ^9: resolved to 10.6.0

These packages were resolved to a version that does not satisfy a peer dependency constraint, which can cause runtime errors. To silence this warning, install a compatible version or add an "overrides" entry to your package.json or deno.json.
Learn more: https://docs.deno.com/go/peer-deps
```

**Comment by ljharb** [comment]
yes, i'm aware; the peer deps mean it's not yet compatible, so it's an error to try to use them together.
